app/coffee/Notifications.coffee

```diff
@@ -22,25 +22,36 @@ module.exports =
 				logger.log number:number, user_id:user_id, key:notification.key, "alredy has notification key for user"
 				callback number
 			else
-				doc = 
+				doc =
 					user_id: ObjectId(user_id)
 					key: notification.key
 					messageOpts: notification.messageOpts
 					templateKey: notification.templateKey
+					expires: notification.expires || false
+				# ttl index on `expiresFrom` field
+				if doc.expires
+					doc.expiresFrom = new Date()
```

Comment:
assume we can't put the expire index on the doc.expires? if so little comment explaining why we have doc.expires and doc.expiresFrom would be good